sky/serve/serve_state.py

```diff
@@ -253,8 +259,7 @@ def set_service_load_balancer_port(service_name: str,
 
 def _get_service_from_row(row) -> Dict[str, Any]:
     (name, controller_job_id, controller_port, load_balancer_port, status,
-     uptime, policy, auto_restart, requested_resources,
-     requested_resources_str) = row[:10]
+     uptime, policy, requested_resources, requested_resources_str) = row[:9]
```

Comment:
```suggestion
     uptime, policy, _, requested_resources, requested_resources_str) = row[:10]
```
and we could just discard the column's value here